demo-shell/src/app/components/cloud/shared/cloud-settings.component.ts

```diff
@@ -69,9 +69,6 @@ export class CloudSettingsComponent implements OnInit, OnDestroy {
     setCurrentSettings(settings) {
         if (settings) {
             this.multiselect = settings.multiselect;
-            this.actionMenu = this.actionMenu;
```

Comment:
i guess its a mistake, do not remove just change `this` to `settings`.
 ```suggestion
            this.actionMenu = settings.actionMenu;
            this.contextMenu = settings.contextMenu;
            this.actions = settings.actions;```